docs/12_0_0/components/datatable.md

```diff
@@ -331,6 +331,11 @@ When a custom component is used as a filter facet, filtering needs to be called
 preferred event such as `onchange="PF('carsTable').filter()"`. Also defining a converter might be
 necessary if the value of the filter facet is not defined.
 
+Please make sure that the filter is using the **same type as the column field** if you are using comparable
```

Comment:
@melloware can you please check the wording? If okay, I'll add this paragraph to all versions.

Reply:
Looks good to me but maybe make comparable like `java.lang.Comparable` just so it is clear.